Command/AlignCommand.php

```diff
@@ -277,7 +277,8 @@ private function checkTarget(InputInterface $input)
             return;
         }
 
-        $target = $this->environmentService->getAliasByName($targetName);
+        $this->environmentService->clearCache();
```

Comment:
Is not possible to update the cache on create a new environment (for a future refactoring)